bookkeeper-server/src/main/java/org/apache/bookkeeper/util/collections/ConcurrentLongHashMap.java

```diff
@@ -226,6 +226,12 @@ public V computeIfAbsent(long key, LongFunction<V> provider) {
         return getSection(h).put(key, null, (int) h, true, provider);
     }
 
+    public V computePut(long key, LongFunction<V> provider, boolean onlyIfAbsent) {
```

Comment:
Why do we need this public interface?

Reply:
the ```computePut```  is similar to ``` computeIfAbsent``` ,  computePut is currently only used for testing. 


> Found another bug, when the onlyIfAbsent of `org.apache.bookkeeper.util.collections.ConcurrentLongHashMap.Section#put` is false, the following unit test fails, I will fix it together: ![image](https://user-images.githubusercontent.com/39521534/208384700-2a617d23-32a4-440b-8e38-959e2625d793.png)
> 
> ![image](https://user-images.githubusercontent.com/39521534/208384650-b7be917b-a00c-40fd-a90b-ec65b4303cbb.png)